CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unrealeased]
 
+## [2.20.0]
```

Comment:
```suggestion
## [2.20.0] - 2023-02-17
```
I can see it hasn't been done since the 2.15.0 but it's usually a standard to add the date there

Reply:
It's fixed ! Good catch